Week2/big-data.js

```diff
@@ -0,0 +1,59 @@
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
```

Comment:
could you change the user from user to 'hyfuser' and password to 'hyfpassword' ?

Reply:
yes you are right done !